pkg/resource/sitewise.go

```diff
@@ -28,7 +28,7 @@ func (rp *SitewiseResources) Asset(ctx context.Context, assetId string) (*iotsit
 }
 
 func (rp *SitewiseResources) Property(ctx context.Context, assetId string, propertyId string, propertyAlias string) (*iotsitewise.DescribeAssetPropertyOutput, error) {
-	if propertyAlias != "" {
```

Comment:
Might be good to separate out changes to the codebase from refactoring tests wdyt?

Reply:
Extracted changes here #192. Once that one is merged, I'll merge this PR. Temporarily switched the base branch from `main` for this PR to #192 so this only shows the refactoring of the tests.